tests/test.h

```diff
@@ -1,11 +1,25 @@
 #ifndef TEST_H
 #define TEST_H
 
+#include <regex.h> 
+#include <string.h>
```

Comment:
You dont need to include string.h since thats already included in the clients below